src/Input/input.style.js

```diff
@@ -1,44 +1,71 @@
 import styled from "styled-components";
 
 const InputStyles = styled.div`
+  width: 100%;
+
   label {
+    display: flex;
+    flex-direction: column;
     color: #353f41;
     font-size: 16px;
     font-weight: 500;
-    font-style: normal;
-    font-stretch: normal;
-    line-height: 2.5;
+    letter-spacing: normal;
+    line-height: 1.38;
   }
 
   input {
-    width: 100%;
     height: 48px;
     border-radius: 2px;
     background-color: #ffffff;
     border: solid 1px #cdcdcd;
-    font-size: 16px;
+    font-size: 18px;
     font-weight: normal;
-    font-style: normal;
-    font-stretch: normal;
     color: #353f41;
     padding: 2px 16px;
     transition: border-color 0.2s ease, box-shadow 0.2s ease;
-  }
 
-  input:focus {
-    outline: none;
-    border-color: #00bfac;
-    transition: border-color 0.2s ease, box-shadow 0.2s ease;
+    &:hover {
+      border-color: #04d4bf;
+      transition: border-color 0.2s ease;
+    }
+
+    &:focus {
+      outline: none;
+      box-shadow: 0 0 0 2px rgba(4, 212, 191, 0.2);
+      border-color: #04d4bf;
+      transition: border-color 0.2s ease, box-shadow 0.2s ease;
+    }
+
+    &::placeholder {
+      color: #bfbfbf;
+    }
   }
 
-  input::placeholder {
-    color: #cdcdcd;
+  .invalid {
+    border-color: #e94119;
```

Comment:
i see we kinda using this color quite frequently here. mb add it in the `Colors` js file we have and import it from there